src/prefect/tasks/gsheets/gsheets.py

```diff
@@ -0,0 +1,105 @@
+from prefect.utilities.tasks import defaults_from_attrs
+import gspread
+from typing import Any, Callable, List, Union
+from prefect import task, Task

```

Comment:
Per the [static analysis tests](https://app.circleci.com/pipelines/github/PrefectHQ/prefect/4688/workflows/9d20cd79-99a1-46bd-98c9-66d37163edc1/jobs/85669/steps) , you have a few extraneous imports.

```suggestion
from typing import Any, List, Union
from prefect import Task
```